### PR TITLE
Automated Test - COMMERCE-5912 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**

### CODE REVIEW

**Comment by LucasAcacio** [comment]
https://issues.liferay.com/browse/COMMERCE-5912, cc:@lorenzolapescara

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
Review started

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceCompare.testcase [hunk 1]
```diff
@@ -27,13 +27,39 @@ definition {
...
+
+		Click(
+			key_navUnderline = "Members",
+			locator1 = "NavUnderline#NAV_UNDERLINE_LINK");
 
+		CommerceAcceleratorsInitializer.inviteNewMemberWithRoleToAccount(
+			emailAddress = "buyer@liferay.com",
+			role = "Buyer");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceCompare.testcase:52]: The test does not include these steps.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceCompare.testcase [hunk 2]
```diff
@@ -27,13 +27,39 @@ definition {
...
+
+		CommerceAcceleratorsInitializer.gotoMiniumProductDetailsPage(productName = "U-Joint");
+
+		Click.clickAtNotVisible(
+			checkboxName = "Compare",
+			locator1 = "Checkbox#ANY_CHECKBOX");
+
+		AssertElementPresent(
+			key_imageFileName = "u-joint",
+			locator1 = "PageEditor#CONTENT_PAGE_IMAGE_CONTENT");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceCompare.testcase:62]:
```suggestion
		AssertElementPresent(locator1 = "CommerceEntry#MINI_COMPARE_ITEM_ACTIVE");
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by LucasAcacio** [comment]
Updated.

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
@LucasAcacio please remove `@ignore = "true"` from the test.

**Comment by lorenzolapescara** [comment]
ci:test:st

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:stop

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:forward

**Pull request closed without merging**